src/org/opensolaris/opengrok/web/WebappListener.java

```diff
@@ -87,6 +94,24 @@ public void contextInitialized(final ServletContextEvent servletContextEvent) {
                 }
             }
         }
+
+        String pluginDirectory = context.getInitParameter("authorizationPluginDirectory");
+        if (pluginDirectory != null) {
+            env.getConfiguration().setPluginDirectory(pluginDirectory);
+            AuthorizationFramework.getInstance(); // start + load
+        } else {
+            if (env.getDataRootPath() == null) {
+                env.getConfiguration().setPluginDirectory(PLUGIN_DIRECTORY_DEFAULT);
+            } else {
+                env.getConfiguration().setPluginDirectory(env.getDataRootPath() + "/../" + PLUGIN_DIRECTORY_DEFAULT);
+            }
+            LOGGER.log(Level.INFO, "authorizationPluginDirectory is not set in web.xml. Default location will be used.");
+        }
+
+        String watchDog = context.getInitParameter("enableAuthorizationWatchDog");
```

Comment:
I think the new params can also be constants
do we describe/document all the params in README or docs? (I think we describe some) Can we add there also these 2 new ones (and we perhaps lack the host:port for config listener)?